lib/src/nav-tabs/Tab.tsx

```diff
@@ -93,7 +93,7 @@ const TabContainer = styled.div<{ active: TabProps["active"] }>`
   align-items: stretch;
   border-bottom: 2px solid ${(props) => (props.active ? props.theme.selectedUnderlineColor : 'transparent')};
   padding: 0.5rem;
-
+  z-index: 1;
```

Comment:
There's no need for this change.

Reply:
If we don't give the Tabs a higher z-index, the color of the `Underline` and the `border-bottom` of the Tab will be overlapped right? (Showing only the gray color)

Reply:
You are right!